internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/TestFabricBuildout.java

```diff
@@ -29,9 +27,17 @@
 import org.testng.Assert;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Ignore;
 import org.testng.annotations.Test;
 
 
+/**
+ * Note: These tests use an unsupported mode during setup - by creating stores and running pushes
+ * to them directly in child regions. This is not how new regions will be added. We need the test setup to support
+ * adding blank regions so that we can simulate the true fabric buildout process. Because of this, I've disabled these
+ * tests for now.
+ */
+@Ignore
```

Comment:
Can you create a issue in github or JIRA and attach the todo?

Reply:
https://github.com/linkedin/venice/issues/1082